upcloud/service/load_balancer_test.go

```diff
@@ -1039,7 +1039,7 @@ func TestLoadBalancerPage(t *testing.T) {
 		net, err := createLoadBalancerAndPrivateNetwork(ctx, svc, zone, "172.16.0.0/24")
 		require.NoError(t, err)
 		lbs := make([]*upcloud.LoadBalancer, 0)
-		for i := 0; i < 5; i++ {
+		for range 5 {
```

Comment:
Just noting that this same change is in #397. Some rebase action needed depending on which gets merged first.